### PR TITLE
fix: always use roles passed to contract for motions

### PR DESCRIPTION
**Pull request by bassgeta**
Instead of getting the "delta" state and applying just the difference in roles in our database, we should just use whatever the FE passed to the contracts and apply it.
[CDApp PR](https://github.com/JoinColony/colonyCDapp/pull/3490)

### CODE REVIEW

**Review by iamsamgibbs — APPROVED** [review]
Reviewed with the CDapp counterpart. All working as expected and nice simplification of the code here now.

**Review by mmioana — APPROVED** [review]
Nice and much easier to follow through with your changes @bassgeta Tested as part of the [CDapp PR](https://github.com/JoinColony/colonyCDapp/pull/3490#pullrequestreview-2401501784)

**Review by rdig — APPROVED** [review]
Tested as part of JoinColony/colonyCDapp#3490 

All working as intended. Nicely fixed :100:

File: src/handlers/multiSig/multiSigCreated/handlers/setUserRoles.ts [hunk 1]
```diff
@@ -20,22 +17,12 @@ export const handleSetUserRolesMultiSig = async (
   }
   // When setting 'Own' authority, the action target will be the colonyAddress
   // When setting 'Multisig' authority, the action target will be the multisig extension address
-  const isMultiSig = actionTarget !== colonyAddress;
+  const isMultiSig = actionTarget !== constants.AddressZero;
```
Comment (iamsamgibbs) [inline src/handlers/multiSig/multiSigCreated/handlers/setUserRoles.ts:20]: Oh, good catch! Can you update the comment here too?

**Pull request merged**